src/components/colorbar/draw.js

```diff
@@ -0,0 +1,533 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var d3 = require('d3');
+
+var Plotly = require('../../plotly');
+
+var attributes = require('./attributes');
+
+
+module.exports = function draw(gd, id) {
```

Comment:
`Plotly.Colorbar` used to be a function with `.attributes` and `.supplyDefaults` attached to it which was rather confusing.

Now, the main `Colorbar` method is call `draw`.